webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.html

```diff
@@ -0,0 +1,87 @@
+<!--View sync message overall status-->
+<div>
```

Comment:
I'm not sure how this will look in the UI but I'm guessing it won't look right, I would think the search mechanism should be in hooked plugged into the datatables library we are using to render the items and your search replaces that of the datatables.

Reply:
Do you mind sending on slack the screenshot of this page?

Reply:
Done